src/palladio/AnnotationParsing.cpp

```diff
@@ -239,32 +239,47 @@ AttributeTrait detectAttributeTrait(const prt::Annotation* annotation) {
 }
 
 AttributeAnnotationInfo getAttributeAnnotationInfo(const std::wstring& attributeName, const RuleFileInfoUPtr& info) {
-	AttributeAnnotationInfo attributeAnnotationInfo;
+	const prt::Annotation* annotation = nullptr;
+	AttributeTrait attributeTrait = AttributeTrait::NONE;
+	std::wstring description = L"";
```

Comment:
redundant initialization, remove the L"";

Reply:
done